MBBSEmu/CPU/CPUCore.cs

```diff
@@ -2051,8 +2051,8 @@ private void Op_Test_8()
             var source = GetOperandValueUInt8(_currentInstruction.Op1Kind, EnumOperandType.Source);
 
             destination &= source;
-            Registers.F.Evaluate(EnumFlags.ZF, destination);
-            Registers.F.Evaluate(EnumFlags.SF, destination);
+            Flags_EvaluateZero(destination);
```

Comment:
these always seem to come in pairs, perhaps combine the zero/sign evaluation into one method call for some minor gains?

Reply:
I reviewed every opcode which modifies both the sign or the zero flag and they're always in tandem. I checked in the modified method which evaluates the value and sets both flags accordingly.